templates/http/modules.j2

```diff
@@ -350,5 +350,37 @@ http2_recv_buffer_size {{ http2['recv_buffer_size'] }};
 {% if http2['chunk_size'] is defined %}
 http2_chunk_size {{ http2['chunk_size'] }};
 {% endif %}
+{% endmacro %}
```

Comment:
```suggestion

{% endmacro %}
```
Remember to always add an extra white line before `{% endmacro %}`. It helps with making the config more readable :)